IdentityCore/src/webview/embeddedWebview/ui/MSIDWebviewUIController.h

```diff
@@ -47,6 +48,8 @@ NSWindowController
 @property (nonatomic) UIModalPresentationStyle presentationType;
 #endif
 
++ (WKWebViewConfiguration *)createWebViewConfigWithPKeyAuthUserAgent;
```

Comment:
nit: let's call this something more generic? e.g. defaultWKWebViewConfiguration

Reply:
Sure!